docs/Reference/Validation-and-Serialization.md

```diff
@@ -611,6 +611,44 @@ const schema = {
 
 fastify.post('/the/url', { schema }, handler)
 ```
+You can even have a specific response schema for different content types.
```

Comment:
I didn't find any example having the content type directly in the object without the `schema` field, but I might have missed that. could you point me out to where exactly it's?

Aside from that, I've noticed that we can also support body validation based on the content type, I think it's not supported at fastify in the meantime right? 

```JS
       body: {
          'application/json': {
              type: 'object',
              properties: {
                hello: {
                  description: 'hello',
                  type: 'string'
                }
              },
              required: ['hello']
          }
      },
```

What do you think? this can definitely take place in a separate PR.